### PR TITLE
you can now logout

### PR DESCRIPTION
**Pull request by charlesmadere**

### CODE REVIEW

File: app/src/main/kotlin/app/youkai/data/local/Credentials.kt [hunk 1]
```diff
@@ -40,4 +40,12 @@ object Credentials {
...
     val isAuthenticated: Boolean
         get() = username != null && authToken != null
-}
\ No newline at end of file
+
+    fun logout() {
+        authToken = null
+        username = null
+        refreshToken = null
+        userId = null
```
Comment (xiprox) [inline app/src/main/kotlin/app/youkai/data/local/Credentials.kt]: Should we go with `prefs.edit().clear()` instead here? 🤔

Comment (charlesmadere) [reply app/src/main/kotlin/app/youkai/data/local/Credentials.kt]: ✔️

**Comment by charlesmadere** [comment]
please check again :smile

**Comment by xiprox** [comment]
💯  Merging...

**Pull request merged**